pkgs/tools/video/rav1e/default.nix

```diff
@@ -13,7 +13,15 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1xaincrmpicp0skf9788w5631x1hxvifvq06hh5ribdz79zclzx3";
 
-  nativeBuildInputs = [ nasm ];
+  nativeBuildInputs = [ nasm cargo-c ];
+
+  postBuild = ''
+    cargo cbuild --release --frozen --prefix=${placeholder "out"}
+  '';
+
+  postInstall = ''
+    cargo cinstall --release --frozen --prefix=${placeholder "out"}
+  '';
```

Comment:
Would be nicer to have this as a hook from `cargo-c` but I don't think we have an extensible abstraction for this yet (though I'm not familiar with our Rust abstractions).

Also: I guess this breaks cross-compilation (if it did work before).